doc/nrf/config_and_build/dfu/nrf70_fw_patch_update.rst

```diff
@@ -0,0 +1,194 @@
+   It includes both the MCUboot header and the nRF70 Series firmware patch.
+
+   For example, the MX25R64 device has a sector size of 4 kB, so the following configuration can be used:
+
+    .. code-block:: console
+
+        mcuboot_primary_2:
+            orig_span: &id003
+            - nrf70_wifi_fw_mcuboot_pad
+            - nrf70_wifi_fw
+            span: *id003
+            address: 0x12F000
+            size: 0x21000
+            device: MX25R64
+            region: external_flash
+
+#. Create the ``mcuboot_secondary_X`` partition for MCUboot, where ``X`` represents the appropriate partition number as described in the :ref:`nrf70_fw_patch_update_mcuboot_partitions` section.
+
+   This partition should start at the address immediately following the end of the ``mcuboot_primary_X`` partition and have the same size as the primary partition.
+   This partition will be used to store the new nRF70 Series firmware patch during the DFU procedure.
+
+   For example:
+
+    .. code-block:: console
+
+        mcuboot_secondary_2:
+            address: 0x150000
+            size: 0x21000
+            device: MX25R64
+            region: external_flash
+
+#. Update the ``external_flash`` partition to allocate all available memory space to it.
+
+   For example:
+
+    .. code-block:: console
+
+        external_flash:
+            address: 0x171000
+            size: 0x68F000
+            device: MX25R64
+            region: external_flash
+
+.. note::
+    Please note that the actual configuration syntax for the Partition Manager will depend on the specific system and tools being used.
+    The example provided is for illustrative purposes and may need to be adjusted to fit the actual configuration file format and syntax required by the Partition Manager in use.
+
+Configuring build system
+------------------------
+
+To enable the DFU procedure for the nRF70 Series firmware patch, complete the following steps depending on the platform:
+
+* For the nRF5340 DK without the Network Core:
+
+    1. Set the :kconfig:option:`CONFIG_NRF_WIFI_FW_PATCH_DFU`` Kconfig option to ``y``.
+    #. Set the :kconfig:option:`CONFIG_UPDATEABLE_IMAGE_NUMBER`` Kconfig option to ``2``.
+    #. For the MCUBoot child image, set the :kconfig:option:`CONFIG_UPDATEABLE_IMAGE_NUMBER`` Kconfig option to ``2``.
+
+* For the nRF5340 DK with the Network Core:
+
+    1. Set the :kconfig:option:`CONFIG_NRF_WIFI_FW_PATCH_DFU`` Kconfig option to ``y``.
+    #. Set the :kconfig:option:`CONFIG_UPDATEABLE_IMAGE_NUMBER`` Kconfig option to ``3``.
+    #. For the MCUBoot child image, set the :kconfig:option:`CONFIG_UPDATEABLE_IMAGE_NUMBER`` Kconfig option to ``3``.
+
+For example, to build the sample with the DFU procedure for the nRF70 Series firmware patch on the nRF5340 DK platform, which includes the network core image, run the following commands:
```

Comment:
Just FYI, I noticed some inconsistencies (not just in your doc) on capitalisation of "Application/Network core". Am looking into this, and will get back to you on which to use.